modules/ROOT/pages/tutorials/nbm-selection-1.adoc

```diff
@@ -315,82 +318,81 @@ You now have a skeleton `TopComponent` - a singleton component called `MyViewerT
+`Utilities.actionsGlobalContext()`. Instead of tracking focus yourself, you can listen to this global selection
+`Lookup`, which fires changes whenever focus changes.
+
+
+3. Edit the `MyViewerTopComponent` source to add a lookup result member and implement the `componentOpened`,
+`componentClosed`, and `resultChanged` methods:
++
 [source,java]
 ----
-    private Lookup.Result<Event> result = null;
+private Lookup.Result<Event> result = null;
 
-    @Override
-    public void componentOpened() {
-        result = Utilities.actionsGlobalContext().lookupResult(Event.class);
-        result.addLookupListener (this);
-    }
+@Override
+public void componentOpened() {
+    result = Utilities.actionsGlobalContext().lookupResult(Event.class);
+    result.addLookupListener (this);
+}
 
-    @Override
-    public void componentClosed() {
-        result.removeLookupListener(this);
-    }
+@Override
+public void componentClosed() {
+    result.removeLookupListener(this);
+}
 
-    @Override
-    public void resultChanged(LookupEvent lookupEvent) {
-        Collection<? extends Event> allEvents = result.allInstances();
-        if (!allEvents.isEmpty()) {
-            Event event = allEvents.iterator().next();
-            jLabel1.setText(Integer.toString(event.getIndex()));
-            jLabel2.setText(event.getDate().toString());
-        } else {
-            jLabel1.setText("[Nothing selected]");
-            jLabel2.setText("");
-        }
+@Override
+public void resultChanged(LookupEvent lookupEvent) {
+    Collection<? extends Event> allEvents = result.allInstances();
+    if (!allEvents.isEmpty()) {
+        Event event = allEvents.iterator().next();
+        jLabel1.setText(Integer.toString(event.getIndex()));
+        jLabel2.setText(event.getDate().toString());
+    } else {
+        jLabel1.setText("[Nothing selected]");
+        jLabel2.setText("");
     }
+}
 ----
++
+* `componentOpened()` is called whenever the component is made visible by the window system; `componentClosed()` is
+called whenever the user clicks the X button on its tab to close it. So whenever the component is showing, you want it
+to be tracking the selection - which is what the above code does.
+* The `resultChanged()` method is your implementation of `LookupListener`. Whenever the selected `Event` changes, it
+will update the two `JLabel`s you put on the form.
 
-* `componentOpened()` is called whenever the component is made visible by the window system; `componentClosed()` is called whenever the user clicks the X button on its tab to close it. So whenever the component is showing, you want it to be tracking the selection - which is what the above code does.
-* The `resultChanged()` method is your implementation of `LookupListener`. Whenever the selected `Event` changes, it will update the two `JLabel`s you put on the form.
-
++
 The required import statements for the `MyViewerTopComponent` are as follows:
```

Comment:
nitpick: might be on the wrong indentation level.  Its currently aligned with the bullet list entries. Move one to the left?